src/Resources/views/CRUD/display_percent.html.twig

```diff
@@ -0,0 +1,19 @@
+{#
+
+This file is part of the Sonata package.
+
+(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+
+For the full copyright and license information, please view the LICENSE
+file that was distributed with this source code.
+
+#}
+
+{%- apply spaceless %}
+    {%- if value is null -%}
+        &nbsp;
+    {%- else -%}
+        {% set value = value * 100 %}
+        {{ value }} %
+    {%- endif -%}
```

Comment:
Why a nbsp ?

```suggestion
    {%- if value is not null -%}
        {{ value * 100 }} %
    {%- endif -%}
```
Should be enough ?

Reply:
Years ago \&nbsp was necessary to prevent problems with cells, but I don't think they are needed anymore. I used the same code as "display_datetime", "display_date" and "display_time" where also \&nbsp is displayed when value is empty. Do you want me to change those too?

Reply:
If we don't need them, let's avoid them indeed.
Elsewhere, but if it's for consistency, I won't block this PR.